src/components/input/Input.tsx

```diff
@@ -2,7 +2,7 @@ import { Input as AntInput, Form } from "antd";
 import { ComponentProps } from "react";
 import { useController } from "react-hook-form";
 
-type TInput<G> = Omit<ComponentProps<"input">, "size" | "ref" | "name"> & {
+export type TInput<G> = Omit<ComponentProps<"input">, "size" | "ref" | "name"> & {
```

Comment:
We need to rename this to form input and create a dummy input in seperate file and create storybook base on that

Reply:
done!